commit 72e8fdd603d40c36bab240c3bcc8e43a3d79c6f2
Author: davitacols <davitacols@gmail.com>
Date:   Tue Mar 24 14:30:10 2026 +0100

    Fixed #36976 -- Made admin action counter a live region for screen readers.

diff --git a/django/contrib/admin/templates/admin/actions.html b/django/contrib/admin/templates/admin/actions.html
index db7345dc60..ff2a5fe3d7 100644
--- a/django/contrib/admin/templates/admin/actions.html
+++ b/django/contrib/admin/templates/admin/actions.html
@@ -9,7 +9,7 @@
     {% endblock %}
     {% block actions-counter %}
     {% if actions_selection_counter %}
-        <span class="action-counter" data-actions-icnt="{{ cl.result_list|length }}">{{ selection_note }}</span>
+        <span class="action-counter" data-actions-icnt="{{ cl.result_list|length }}" aria-live="polite" aria-atomic="true">{{ selection_note }}</span>
         {% if cl.result_count != cl.result_list|length %}
         <span class="all hidden">{{ selection_note_all }}</span>
         <span class="question hidden">
